react/Viewer/Readme.md

```diff
@@ -48,20 +55,48 @@ const files = [
 
 // The host app will usually need a small wrapper to display the Viewer. This is a very small example of such a wrapper that handles opening, closing, and navigating between files.
 initialState = {
-  viewerOpened: false,
+  viewerOpened: isTesting(),
   currentFileIndex: 0,
-  close: true
+  toolbarClose: true
```

Comment:
Nit: I think it would be a bit simpler to use a positive variable here, with a past tense verb denoting that it is referring to a state, not to a method/function, with the same name as the prop.

toolbarClose -> panelOpened.